jsguide.html

```diff
@@ -2825,23 +2825,23 @@ <h3 id="jsdoc-enum-and-typedef-comments">7.7 Enum and typedef comments</h3>
 also have a description. Individual enum items may be documented with a JSDoc
 comment on the preceding line.</p>
 
-<pre><code class="language-js prettyprint good">/**
- * A useful type union, which is reused often.
- * @typedef {!Bandersnatch|!BandersnatchType}
- */
-let CoolUnionType;
-
-
-/**
- * Types of bandersnatches.
- * @enum {string}
- */
-const BandersnatchType = {
-  /** This kind is really frumious. */
-  FRUMIOUS: 'frumious',
-  /** The less-frumious kind. */
-  MANXOME: 'manxome',
-};
+<pre><code class="language-js prettyprint good">
+  /**
```

Comment:
the indentation here is incorrect.  you can see it when you view the final rendered page.

leave the `/**` on the previous line with the `<pre>` HTML tags, and delete this one.